### PR TITLE
Clean up test extensions for UI tests

### PR DESCRIPTION
**Pull request by rachelmcr**
This removes unused test extensions from a past implementation of our UI tests. These are now covered in the page objects under `/Screens`, aside from the shared `isIpad` function that can now be called from within the page objects.

To test:

* Start the simulator.
* Confirm all of the tests in WordPressUITests still pass.

### CODE REVIEW

File: WordPress/WordPressUITests/Utils/XCTest+Extensions.swift [hunk 1]
```diff
@@ -59,156 +20,15 @@ extension XCUIElement {
...
     func isIPhone() -> Bool {
         let app = XCUIApplication()
         return app.windows.element(boundBy: 0).horizontalSizeClass == .compact || app.windows.element(boundBy: 0).verticalSizeClass == .compact
     }
 }
+
+func isIpad() -> Bool {
+    let app = XCUIApplication()
+    return app.windows.element(boundBy: 0).horizontalSizeClass == .regular && app.windows.element(boundBy: 0).verticalSizeClass == .regular
+}
```
Comment (jkmassel) [inline WordPress/WordPressUITests/Utils/XCTest+Extensions.swift]: Saw this posted, and figured I'd add a drive-by comment.

Does using:

```
return UIDevice.current.userInterfaceIdiom == .pad
```

work for this case? I think it _should_, given that it's running on-device?

Just thinking that there's no guarantee the size class will correspond to a specific device type, so depending what you're using this for, it might be nice to let Apple do the heavy lifting here?

Comment (rachelmcr) [reply WordPress/WordPressUITests/Utils/XCTest+Extensions.swift]: Good call! That should do the trick. I was just reusing the same approach that had been implemented in the tests before, but this is more robust. Will update to use this. Thanks!

Comment (jkmassel) [reply WordPress/WordPressUITests/Utils/XCTest+Extensions.swift]: Should've mentioned this before, too 😅 

Assuming everything still works the way you want, you could use 

```
return UIDevice.current.userInterfaceIdiom == .phone
```

for the other one 🙃

File: WordPress/WordPressUITests/Utils/XCTest+Extensions.swift [hunk 2]
```diff
@@ -59,156 +20,14 @@ extension XCUIElement {
...
-        self.typeText(deleteString)
-        self.typeText(text)
-    }
-}
-
 extension XCTest {
 
     func isIPhone() -> Bool {
         let app = XCUIApplication()
         return app.windows.element(boundBy: 0).horizontalSizeClass == .compact || app.windows.element(boundBy: 0).verticalSizeClass == .compact
```
Comment (JavonDavis) [inline WordPress/WordPressUITests/Utils/XCTest+Extensions.swift]: Just a reminder about @jkmassel  comment for this one as well

```suggestion
        return UIDevice.current.userInterfaceIdiom == .phone
```

File: WordPress/WordPressUITests/Utils/XCTest+Extensions.swift (hunk 1 above)
Comment (rachelmcr) [reply WordPress/WordPressUITests/Utils/XCTest+Extensions.swift]: Hah, yeah, thanks! I forgot that was in there. :)
